Src/IdentityService.WebAPI/V1/Controller/AuthorizationController.cs

```diff
@@ -23,8 +23,12 @@ public class AuthorizationController : ControllerBase {
 		this._logger = logger;
 	}
 
+	/// <summary>
+	/// Authorizes a user which requested to do something requiring authorization
```

Comment:
'Validates the provided token and the existence of the user ID within the token Sub.' Hat nichts mit einer Request zu tun. Kann auch unabhängig aufgerufen werden